src/api-calls.js

```diff
@@ -0,0 +1,11 @@
+const fetchBookData = () => {
```

Comment:
Great work breaking out your network requests into its own file and including error handling!  Cheers! 🎉